modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/SnapshotMetadata.java

```diff
@@ -63,6 +63,9 @@ public class SnapshotMetadata implements Serializable {
      */
     private final String folderName;
 
+    /** If {@code true} then compress the files. */
+    private final boolean compress;
```

Comment:
There is already `compGrpIds`. But its compression differs from dump compression. Let's name this param in different way?